arch/xtensa/src/esp32/esp32_irq.c

```diff
@@ -595,11 +580,11 @@ void up_enable_irq(int irq)
   int cpuint = IRQ_GETCPUINT(g_irqmap[irq]);
 
   DEBUGASSERT(cpuint >= 0 && cpuint <= ESP32_CPUINT_MAX);
-#ifdef CONFIG_SMP
+<<<<<<< HEAD
```

Comment:
Is it a result of unresolved merge conflict?

Reply:
Yes, my fault. Fixed.